Makefile

```diff
@@ -94,6 +96,11 @@ minikube_iso:
 	$(MAKE) BR2_EXTERNAL=../../deploy/iso/minikube-iso minikube_defconfig -C $(BUILD_DIR)/buildroot
 	$(MAKE) -C $(BUILD_DIR)/buildroot
 	mv $(BUILD_DIR)/buildroot/output/images/rootfs.iso9660 $(BUILD_DIR)/minikube.iso
+else
+	docker run --env IN_DOCKER=1 --workdir /mnt --entrypoint /usr/bin/make \
```

Comment:
This codepath technically depends on `buildroot-image`, maybe we can refactor this to include that dependency